mta/fixtures/business_service.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from conftest import generate_description, generate_name
+from swagger_client.models.api_business_service import ApiBusinessService
+
+
+@pytest.fixture()
```

Comment:
Please set the fixture's scope as `session`

Reply:
Done.